src/libYARP_dev/src/devices/RemoteControlBoard/RemoteControlBoard.cpp

```diff
@@ -2067,29 +2013,22 @@ class yarp::dev::RemoteControlBoard :
      * @param v pointer to storage for the return value
      * @return true/false, upon success/failure (you knew it, uh?)
      */
-    virtual bool getMotorEncoderTimed(int j, double *v, double *t) {
+    virtual bool getMotorEncoderTimed(int j, double *v, double *t)
+    {
         double localArrivalTime = 0.0;
-        bool ret = false;
+        bool ret(false);
```

Comment:
FYI This change has no effect, since copy initialization and direct initialization do exactly the same for builtin types.